docs/cli.md

```diff
@@ -144,3 +159,39 @@ en.wikipedia.org
 en.wikipedia.org
 (---snip---)
 ```
+
+### Performance
+
+Our `adaparse` tool may outperform other popular alternatives. We offer a [collection of
+sets of URLs](https://github.com/ada-url/url-various-datasets) for benchmarking purposes.
+The following results are on a macBook Air 2022 (M2 processor) using LLVM 14. We
+compare against trurl version 0.6 (libcurl/7.87.0).
```

Comment:
Can you add a link to the `trurl` repo here?

Reply:
done